### ISSUE TITLE
Add reported rate per day information chart for country selected

### ISSUE THREAD

**Issue by sdchew** [CSSEGISandData/COVID-19#67]
Appreciate the great work done by this team to provide this useful resource. Is it possible to add a feature to show the reported rate per day to the chart for the country selected? thank you

**Comment by avatorl** [comment]
Probably you'll the feature you're looking for in my report [](http://avatorl.org/covid-19/) based on JHU CSSE data (and other data sources).

**Comment by sdchew** [comment]
Yes. Looking for this feature using the data currently in JHU CSSE. For
example, if I click Hong Kong, it would be great if the plot on the lower
right can show the trend in HK.

Thank you

[image: image.png]

**Comment by HyperBI** [comment]
You could also check out my report where i have something like you suggest. See link: https://bit.ly/39GuMpP 

You can select countrys or provinces in the right side and will see a timeline on the left side. You can also switch between confirmed cases, deaths and recoverd. 

That will sometimes show you weird graphs like Japan, where it rises, goes down again for a few days and then rises again. It is very difficult right now to say someting about the quality of the data.
